include/ibamr/IBFEMethod.h

```diff
@@ -617,6 +615,35 @@ class IBFEMethod : public IBStrategy
     void setupTagBuffer(SAMRAI::tbox::Array<int>& tag_buffer,
                         SAMRAI::tbox::Pointer<SAMRAI::mesh::GriddingAlgorithm<NDIM> > gridding_alg) const override;
 
+    /*!
+     * Inactivate a structure/part. See IBAMR::IBStrategy::inactivateLagrangianStructure().
+     *
+     * @note Since this class assumes that structures live on the finest grid
+     * level the second argument is ignored.
+     */
+    virtual void inactivateLagrangianStructure(int structure_number = 0,
+                                               int level_number = std::numeric_limits<int>::max()) override;
+
+    /*!
+     * Activate a previously inactivated structure/part to be used again in
+     * FSI calculations. See IBAMR::IBStrategy::activateLagrangianStructure().
+     *
+     * @note Since this class assumes that structures live on the finest grid
+     * level the second argument is ignored.
+     */
+    virtual void activateLagrangianStructure(int structure_number = 0,
+                                             int level_number = std::numeric_limits<int>::max()) override;
+
+    /*!
+     * Determine whether or not the given structure or part is currently
+     * activated. See IBAMR::IBStrategy::getLagrangianStructureIsActivated().
+     *
+     * @note Since this class assumes that structures live on the finest grid
+     * level the second argument is ignored.
+     */
+    virtual bool getLagrangianStructureIsActivated(int structure_number = 0,
+                                                   int level_number = std::numeric_limits<int>::max()) const override;
+
```

Comment:
Could we also go ahead and add some methods to keep track of the "names" of parts and relate them to part numbers? This functionality is provided in `LDataManager` to make it easier to manage activating/inactivating IB parts, and might be useful in general.

Reply:
At some point we need to actually implement `class Part {}` which should include such functionality. Could we just open a separate issue to track that instead of adding such features to this PR?

Reply:
Sure thing.